releasewarrior/wiki_data.py

```diff
@@ -419,6 +419,11 @@ def log_release_status(release, logger):
     for graph_info in release["inflight"][current_build_index]["graphids"]:
         logger.info("%s graph: https://tools.taskcluster.net/task-group-inspector/#/%s",
                     graph_info[0], graph_info[1])
+    logger.info("")
+    logger.info("\treleaserunner variables:")
+    for graph_info in release["inflight"][current_build_index]["graphids"]:
+        logger.info("\t\texport %s_TASK_ID=%s", graph_info[0].upper(), graph_info[1])
```

Comment:
Nit: I'd use `print()` instead of `logger.info()`. This helps to copy and paste the 3 lines at once. I tested it locally, this makes:

```
INFO: RELEASE: devedition 62.0b16 build1 2018-08-09
INFO: promote graph: https://tools.taskcluster.net/task-group-inspector/#/WQJADDasS3qWVuHyvvt0Uw
INFO: push graph: https://tools.taskcluster.net/task-group-inspector/#/V6LnWJIrSQ2ypqyDC9bgQw
INFO: ship graph: https://tools.taskcluster.net/task-group-inspector/#/CNRWluXXSfuoj3risLPDcg
INFO: 
INFO: 	releaserunner variables:
INFO: 		export PROMOTE_TASK_ID=WQJADDasS3qWVuHyvvt0Uw
INFO: 		export PUSH_TASK_ID=V6LnWJIrSQ2ypqyDC9bgQw
INFO: 		export SHIP_TASK_ID=CNRWluXXSfuoj3risLPDcg
[...]
```

into 

```
INFO: RELEASE: devedition 62.0b16 build1 2018-08-09
INFO: promote graph: https://tools.taskcluster.net/task-group-inspector/#/WQJADDasS3qWVuHyvvt0Uw
INFO: push graph: https://tools.taskcluster.net/task-group-inspector/#/V6LnWJIrSQ2ypqyDC9bgQw
INFO: ship graph: https://tools.taskcluster.net/task-group-inspector/#/CNRWluXXSfuoj3risLPDcg
INFO: 
INFO: 	releaserunner variables:
		export PROMOTE_TASK_ID=WQJADDasS3qWVuHyvvt0Uw
		export PUSH_TASK_ID=V6LnWJIrSQ2ypqyDC9bgQw
		export SHIP_TASK_ID=CNRWluXXSfuoj3risLPDcg
```

Reply:
I'm not asking to change this back, but most terminals I've used allow ctrl + shift to highlight ranges (e.g. not even copying the INFO part of the lines)

e.g. (from unrelated code, but shows the shell feature)
![image](https://user-images.githubusercontent.com/1415602/44416164-31fdb280-a540-11e8-8740-1f2e2da67021.png)

Reply:
Yeah, that was why I originally went with logging, but might as well make it easy. I'm not sure there is a good reason for *any* of it to be prefixed by `INFO: `, but can deal with that latter.